### ISSUE TITLE
Double Speed on Google TV

### ISSUE THREAD

**Issue by thizaum** [Swordfish90/Lemuroid#847]
Hello,

I've installed Lemuroid on my Xperia phone and on my Sony TV (with Android TV).
On the phone, it plays normally.
On the TV, the speed is doubled.
If I connect the phone to the TV with a cable, the speed is halved.
If I stream from the phone, it plays normally (with a delay).
Systems: SNES and Genesis/MD.
Note: when I say double or half speed, it might be 150% and 67%, I'm not sure.
Is this related to refresh rate? What's the fix?

Thanks.

**Comment by thizaum** [comment]
Is this abandoned?

**Comment by david79111** [comment]
No, development is ongoing. Also, this issue was already opened #397
